solidstart-tanstackquery-tailwind-modules/src/utils/onclickOutside.ts

```diff
@@ -0,0 +1,10 @@
+import { onCleanup } from 'solid-js';
+
+export function clickOutside(el: any, accessor: any) {
+  const onClick = (e: any) => {
+    !el.contains(e.target) && accessor()?.();
+  };
+  document.body.addEventListener('click', onClick);
+
+  onCleanup(() => document.body.removeEventListener('click', onClick));
+}
```

Comment:
At least we can type el and e.
Never ever saw accessor before, so I don't know how to type it :-D

```suggestion
export function clickOutside(el: Element, accessor: any) {
  const onClick = (e: MouseEvent) => {
    !el.contains(e.target as unknown as Element) && accessor()?.();
  };
  document.body.addEventListener('click', onClick);

  onCleanup(() => document.body.removeEventListener('click', onClick));
}
```